Makefile

```diff
@@ -25,9 +25,12 @@ export GO_IMAGE = golang:$(shell go version | cut -d ' ' -f 3 | tail -c +3 )
 # This is the default target, build everything:
 all: conduit cmd/algorand-indexer/algorand-indexer go-algorand idb/postgres/internal/schema/setup_postgres_sql.go idb/mocks/IndexerDb.go
 
-conduit: go-algorand
+conduit: go-algorand conduit-docs
 	go generate ./... && cd cmd/conduit && go build -ldflags="${GOLDFLAGS}"
 
+conduit-docs:
+	go install ./cmd/conduit-docs/
```

Comment:
I think you can use `go run ./cmd/conduit-docs` in the go-generate command to avoid needing an install step.

Reply:
are these docs for dev portal or are they readmes staying in the conduit repo?

Reply:
I intend them for the dev portal, but we could use them for our readmes as well--there is obviously a lot of overlap.

Is it bad to use `go install`? I've wondered why we don't have an install target for indexer in our makefile.